### PR TITLE
feat: create a documentation layout for desktops & tablets

### PR DESCRIPTION
**Pull request by wisnie**

### CODE REVIEW

**Review by mmiszy — APPROVED** [review]

**Comment by wisnie** [comment]
I just want to mention that all breakpoints would be moved to css variables tomorrow :+1:

File: docs/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+export const Accordion = memo<AccordionProps>(({ heading, isActive, sections }) => {
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const hasSections = sections && sections.length > 0;
+  const toggleExpanded = useCallback(() => setExpanded((prev) => !prev), []);
+  return (
+    <div className={styles.accordionWrapper}>
+      <header className={styles.accordionHeader} onClick={toggleExpanded}>
+        <h3 className={`${styles.accordionHeading} ${isActive ? styles.accordionHeadingActive : ''}`}>{heading}</h3>
+        {hasSections && (
+          <img
```
Comment (mmiszy) [inline docs/src/components/Accordion/Accordion.tsx:18]: Semantically speaking it shouldn't be an `img` – chevron should be added with CSS
(+ some aria attributes) https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html

File: docs/src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  isActive: boolean;
+  sections?: string[];
+};
+export const Accordion = memo<AccordionProps>(({ heading, isActive, sections }) => {
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const hasSections = sections && sections.length > 0;
+  const toggleExpanded = useCallback(() => setExpanded((prev) => !prev), []);
+  return (
+    <div className={styles.accordionWrapper}>
+      <header className={styles.accordionHeader} onClick={toggleExpanded}>
```
Comment (mmiszy) [inline docs/src/components/Accordion/Accordion.tsx:15]: This should be a button

File: docs/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+type HeaderProps = {
+  toggleMenuOpened: () => void;
+};
+
+export const Header = memo<HeaderProps>(({ toggleMenuOpened }) => {
+  return (
+    <header className={styles.header}>
+      <button className={styles.toggleButton} onClick={toggleMenuOpened}>
+        <span className={styles.srOnly}>Toggle reference sidebar</span>
+        <img className={styles.menuIcon} src="/icons/menu.svg" alt="Menu" />
```
Comment (mmiszy) [inline docs/src/components/Header/Header.tsx:14]: Shouldn't this be in css?

File: docs/src/components/Reference/Reference.module.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  @media screen and (min-width: 55rem) {
+    // padding-top: 2rem;
+    // height: auto;
+    // max-height: calc(100vh - var(--header-height));
+    // transform: translateX(0);
+    // position: sticky;
+    // top: var(--header-height);
+    width: var(--reference-width);
+    // background-color: var(--section-background);
+    // flex-shrink: 0;
```
Comment (mmiszy) [inline docs/src/components/Reference/Reference.module.scss:25]:
```suggestion
    width: var(--reference-width);
```

**Comment by mmiszy** [comment]
@all-contributors please add @wisnie for code

**Comment by mmiszy** [comment]
@all-contributors please add @wisnie for code

**Pull request merged**